### PR TITLE
Support all deploy fields on docker deploy form

### PR DESCRIPTION
**Pull request by ssalinas**
@tpetr

### CODE REVIEW

**Comment by tpetr** [comment]
tested this out in vagrant and it doesn't seem as though the port mappings were persisted into the JSON

**Comment by ssalinas** [comment]
@tpetr I had the portMappings outside docker in the json by accident, fixed now and confirmed that the json persisted on the response

**Comment by tpetr** [comment]
some progress -- now `portMappings` is being set, seemingly only the last item when I add more than one in the UI

**Comment by ssalinas** [comment]
should be good now with ea995d0

File: SingularityUI/app/templates/dockerForms/portMap.hbs [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    </div>
+
+    <div class="form-group required">
+        <label for="host-port-{{ timestamp }}">Container Port</label>
+        <input type="text" id="host-port-{{ tiemstamp }}" class="host-port form-control">
+    </div>
+
+    <div class="form-group">
+        <label for="protocal-{{ timestamp }}">Protocol</label>
+        <input type="text" id="protocol-{{ tiemstamp }}" class="protocol form-control" placeholder="default: tcp">
```
Comment (tpetr) [inline SingularityUI/app/templates/dockerForms/portMap.hbs]: `tiemstamp` looks like a typo

**Comment by ssalinas** [comment]
fixed my typo

**Comment by tpetr** [comment]
i think this needs a little more work -- i created a new deploy and added 2 port mappings, but when i inspected the POST, it only included the last port mapping

**Comment by ssalinas** [comment]
That should have been fixed with ea995d0 , it works when I run it locally and I can't seem to reproduce. Certain type of request maybe?

**Comment by tpetr** [comment]
sorry, was testing this on the wrong branch! LGTM

**Pull request merged**